frontend/test/model/EBEventTest.scala

```diff
@@ -79,10 +79,6 @@ class EBEventTest extends PlaySpecification {
       updateEventWithDescription(ebLiveEvent, "<!-- main-image: https://media.gutools.co.uk/images/e5b35b1f20588172b24960314cbf8e9c8482d3bf?crop=27_0_1800_1080 ! -->")
         .mainImageUrl must beSome(Uri.parse("https://media.gutools.co.uk/images/e5b35b1f20588172b24960314cbf8e9c8482d3bf?crop=27_0_1800_1080"))
     }
-    "should not blow up even if the url is really bizarre" in {
-      updateEventWithDescription(ebLiveEvent, "<!-- main-image: http://test.net/## -->")
-        .mainImageUrl must beNone
-    }
```

Comment:
This test was failing because the `scalaUri` library was updated to the latest version in this PR. As a result, [the code being tested](https://github.com/guardian/membership-frontend/blob/paypal-nvp-and-client-side/frontend/app/model/Eventbrite.scala#L287) no longer gives parse errors for the test url given here, and `mainImageUrl` thus returns `Some` rather than `None`.

The latest version of `Uri.parse` seems to be a lot more lenient, and I cannot find a url that will break the parser (I tried some complete garbage urls and they came back fine). As a result, I can't think of an alteration to the test that will activate the error handling code being tested. Therefore I've removed the test.

@rtyley @rupertbates please let me know if you think this is a bad idea, or if there's a better solution that I've missed 🐳.